projects/plugins/videopress/jetpack-videopress.php

```diff
@@ -4,7 +4,7 @@
  * Plugin Name: Jetpack VideoPress
  * Plugin URI: https://wordpress.org/plugins/jetpack-videopress
  * Description: High quality, ad-free video.
- * Version: 1.4.0
```

Comment:
I'm not sure whether we should update the VideoPress version for these changes. I think that using `tools/project-version.sh -f -u 1.12.4-alpha packages/backup` should do the trick to bump only the backup package.

Reply:
Sometime before, I ran `tools/fixup-project-versions.sh` to fix the versions but unknowingly it also touched Videopress files which later forced me to push a changelog for Videopress as well. Will take a look again if I could revert those.

Reply:
Yeah, I will suggest cleaning it up, rebase/merge from trunk and then try again maybe `tools/project-version.sh -f -u 1.12.4-alpha packages/backup`.

I think the changelogger checks for missing version, considering the trunk as well.

Reply:
Done ✅